api/routes/user_routes.py

```diff
@@ -0,0 +1,67 @@
+from db.queries.assessment_records.queries import create_user_application_association
+from db.queries.assessment_records.queries import get_user_application_associations
+from db.queries.assessment_records.queries import (
+    update_user_application_association as update_user_application_association_db,
+)
+from db.schemas.schemas import AllocationAssociationSchema
+from flask import abort
+from flask import current_app
+from flask import request
+
+
+def get_all_users_associated_with_application(application_id, active=None):
+    associations = get_user_application_associations(application_id=application_id, active=active)
+    if associations:
+        serialiser = AllocationAssociationSchema()
+        return serialiser.dump(associations, many=True)
+
+    current_app.logger.error(f"Could not find any users associated with application {application_id}")
+    abort(404)
+
+
+def get_user_application_association(application_id, user_id):
+    association = get_user_application_associations(application_id=application_id, user_id=user_id)
+
+    if association:
+        serialiser = AllocationAssociationSchema()
```

Comment:
Can we create this  serialiser = AllocationAssociationSchema() once only

Reply:
I'm not sure what you mean? It's only created once in the method context.

Reply:
After the discussion we thought not to change the way